### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by MayaLekova**
Please check if the file structure complies with the MVC idea.

### CODE REVIEW

File: Assignment1/Assignment1/MovieData.swift [hunk 1]
```diff
@@ -31,36 +32,43 @@ extension String {
...
+            print(response.data)     // server data
+            print(response.result)   // result of response serialization
+            
+            if let JSON = response.result.value {
+                self.json = JSON
+                
+                if let jsonObj = self.json.parseJSONString {
+                    if let movieData = jsonObj as? NSDictionary {
+                        if let movieObj = Json4Swift_Base(dictionary: movieData)
+                        {
```
Comment (benSmith1981) [inline Assignment1/Assignment1/MovieData.swift]: To not have lots of if let if let you can do this (better than lots of indentation
```Swift
if let JSON = response.result.value,
     let jsonObj = JSON.parseJSONString,
     let movieData = jsonObj as? NSDictionary,
     let movieObj = Json4Swift_Base(dictionary: movieData){
       //CODE TO ITERATE MovieObj here
}
```

**Review by benSmith1981 — CHANGES_REQUESTED** [review]
Nice, do the one fix...Maybe in future you can do smaller pull requests 17,000 lines is quite a lot!!

File: Assignment1/Assignment1/MovieData.swift (hunk 1 above)
Comment (MayaLekova) [reply Assignment1/Assignment1/MovieData.swift]: Sorry about the length of the review!

**Comment by MayaLekova** [comment]
Closing this because the commits are already merged. Fix is applied in next review.

**Pull request closed without merging**